package.json

```diff
@@ -9,15 +9,17 @@
   "scripts": {
     "postinstall": "scripts/react-native-web.sh && cd desktop && npm install",
     "clean": "react-native clean-project-auto",
-    "android": "npm run check-metro-bundler-port && react-native run-android",
-    "ios": "npm run check-metro-bundler-port && react-native run-ios",
+    "android": "npm run check-metro-bundler-port && scripts/dev.sh && react-native run-android",
+    "ios": "npm run check-metro-bundler-port && scripts/dev.sh && react-native run-ios",
     "ipad": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (12.9-inch) (4th generation)\"",
     "ipad-sm": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (9.7-inch)\"",
     "start": "react-native start",
-    "web": "node web/proxy.js & webpack-dev-server --open --config config/webpack/webpack.dev.js",
+    "web": "concurrently npm:web-proxy npm:web-server",
+    "web-server": "scripts/dev.sh && webpack-dev-server --open --config config/webpack/webpack.dev.js",
+    "web-proxy": "node web/proxy.js",
```

Comment:
Is it possible to just do `scripts/dev.sh && node web/proxy.js & webpack-dev-server --open --config config/webpack/webpack.dev.js` to run the dev script first, then everything else? I didn't test but I assume you did & you found that doesn't work?

Reply:
It's possible 😄 
I deliberately extracted separate scripts because the line became too long 
(Originally it was `bash scripts/set-pusher-suffix.sh && node web/proxy.js & webpack-dev-server --open --config config/webpack/webpack.dev.js`)

I can switch to a how it was before

Reply:
Ok cool cool I'm totally fine with it like this, I think it would be nice to get @roryabraham or @AndrewGable 's opinion on this since I believe they constructed this before or at least they are pretty knowledgeable in the area